autotest/pyscripts/test_ogr2ogr_py.py

```diff
@@ -182,10 +182,7 @@ def test_ogr2ogr_py_5():
 
 def test_ogr2ogr_py_6():
 
-    try:
-        import ogr_pg
-    except:
-        pytest.skip()
+    ogr_pg = pytest.importorskip('ogr_pg')
```

Comment:
when wouldn't this be available?

Reply:
Probably needs a `reason=` argument if it needs to stay

Reply:
Not sure that `importorskip` needs a `reason`.